src/Illuminate/Console/Scheduling/Schedule.php

```diff
@@ -28,19 +29,26 @@ class Schedule
         __call as macroCall;
     }
 
-    const SUNDAY = 0;
+    /** @deprecated Use \Illuminate\Console\Enums\ScheduleOn::SUNDAY instead */
+    const SUNDAY = ScheduleOn::SUNDAY;
```

Comment:
As this is scheduled for the next version, I would take the opportunity to move completely to the new approach, and document these changes.

But this is better to check with one of the maintainers. Maybe it is better to keep those for a smooth transition